seerep-srv/seerep-core-fb/include/seerep-core-fb/core-fb-conversion.h

```diff
@@ -165,6 +175,14 @@ class CoreFbConversion
   static void
   fromFbDataLabelsGeneral(const flatbuffers::Vector<flatbuffers::Offset<seerep::fb::BoundingBox2DLabeled>>* labelsBB2d,
                           std::vector<seerep_core_msgs::LabelWithInstance>& labelWithInstance);
+  /**
+   * @brief converts the BoundingBoxLabeled with instances of the flatbuffer data message to seerep core specific message
+   * @param labelsBB the BoundingBoxLabeled with instances in the flatbuffer data message
+   * @param labelWithInstance the BoundingBoxLabeled with instances in the data message in seerep core format
+   */
+  static void
+  fromFbDataLabelsGeneral(const flatbuffers::Vector<flatbuffers::Offset<seerep::fb::BoundingBoxLabeled>>* labelsBB,
```

Comment:
I don't think we need to rename it, since they serve the same purpose and are just overloaded with different parameters.

Reply:
Well, yes, but not quite. The naming convention in that file is "fromFbData" + whatever part of the data-msg is converted. e.g. `fromFbDataHeader` for the conversion of the Header sub-msgs of the fb data msg.

I think I made the first error by naming the second methode also `fromFbDataLabelsGeneral` instead of `fromFbDataLabelsBb2d` as it would have been consistently.

The third methode should be named `fromFbDataLabelsBb`